recipes/iiboost/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "iiboost" %}
+{% set version = "0.2" %}
+{% set sha256 = "0ecd906b453a6a278651e324352bd6f820b70aa4ca15fa42843bdb5c01ee7079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cbecker/iiboost/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - cmath_fixes.diff
+    - python3.diff
+    - msvc_fixes.diff
+
+build:
+  number: 0
+  skip: True        # [np!=112 or (win and py27)]
+  features:
+    - vc14          # [win and py>=35]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - gcc                   # [linux]
+    - cmake
+    - eigen
+    - numpy x.x
+    - itk       4.11.*
+    - boost     1.63.*
```

Comment:
Another option is to do something like, https://github.com/bluescarni/staged-recipes/pull/1